src/components/switcher/Switcher.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import ReactSwitch from 'react-switch';
+
+import GlobeIcon from 'img/globeIcon.svg';
+import IconEN from 'img/EN.png';
+import IconRU from 'img/RU.png';
+
+import colors from 'variables/colors';
+import style from './Switcher.module.scss';
+
+function Switcher(): JSX.Element {
+  const { i18n } = useTranslation();
+
+  const [languageRu, setLanguageRu] = useState(false);
```

Comment:
Are you really sure it is a good solution?

Reply:
are you about naming ?

Reply:
I asked you about your solution

Reply:
i'll fix that

Reply:
Here you are using isn't flexible solution instead you can use the key for any language